sdl-core/src/main/scala/io/smartdatalake/util/misc/SchemaUtil.scala

```diff
@@ -145,6 +147,19 @@ object SchemaUtil {
     StructType.fromDDL(ddl)
   }
 
+  private def parseInputPath(inputPath: String): String = {
```

Comment:
And again a naming discussion :-)
The method doesn't parse, but just reads the content.
Suggestion: readFromPath?